add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The current code directly uses raw SQL queries and procedural style, which can lead to security vulnerabilities such as SQL injection and makes the code less maintainable.\n\nRecommendations: Use prepared statements to prevent SQL injection. Additionally, consider refactoring the code to follow the MVC pattern. Here's an example using PDO for database interactions:

```php
// Database.php
class Database {
    private $host = "localhost";
    private $db_name = "books_db";
    private $username = "root";
    private $password = "";
    public $conn;

    public function getConnection() {
        $this->conn = null;
        try {
            $this->conn = new PDO("mysql:host=" . $this->host . ";dbname=" . $this->db_name, $this->username, $this->password);
            $this->conn->exec("set names utf8");
        } catch(PDOException $exception) {
            echo "Connection error: " . $exception->getMessage();
        }
        return $this->conn;
    }
}

// Book.php (Model)
class Book {
    private $conn;
    private $table_name = "books";

    public $id;
    public $title;
    public $author;
    public $year;

    public function __construct($db) {
        $this->conn = $db;
    }

    public function create() {
        $query = "INSERT INTO " . $this->table_name . " SET title=:title, author=:author, year=:year";

        $stmt = $this->conn->prepare($query);

        $this->title=htmlspecialchars(strip_tags($this->title));
        $this->author=htmlspecialchars(strip_tags($this->author));
        $this->year=htmlspecialchars(strip_tags($this->year));

        $stmt->bindParam(":title", $this->title);
        $stmt->bindParam(":author", $this->author);
        $stmt->bindParam(":year", $this->year);

        if($stmt->execute()) {
            return true;
        }

        return false;
    }
}

// add_book.php (Controller)
include_once 'Database.php';
include_once 'Book.php';

$database = new Database();
$db = $database->getConnection();

$book = new Book($db);

if ($_SERVER['REQUEST_METHOD'] == 'POST') {
    $book->title = $_POST['title'];
    $book->author = $_POST['author'];
    $book->year = $_POST['year'];

    if ($book->create()) {
        echo "Book added successfully!";
    } else {
        echo "Unable to add book.";
    }
}
```

This approach improves security, maintainability, and follows the Single Responsibility Principle by separating database logic from the application logic.